zebra-state/src/service/non_finalized_state/chain.rs

```diff
@@ -11,25 +11,46 @@ use zebra_chain::{
     transaction::Transaction::*, transparent, work::difficulty::PartialCumulativeWork,
 };
 
-use crate::{service::check, PreparedBlock, ValidateContextError};
+use crate::{service::check, ContextuallyValidBlock, PreparedBlock, ValidateContextError};
 
 #[derive(Debug, Default, Clone)]
 pub struct Chain {
-    pub blocks: BTreeMap<block::Height, PreparedBlock>,
+    /// The contextually valid blocks which form this non-finalized partial chain, in height order.
+    pub(crate) blocks: BTreeMap<block::Height, ContextuallyValidBlock>,
+
+    // chain state
+    //
```

Comment:
Tiny nit: having the comment line merge with the documentation initially made me think that the `chain state` was the documentation header :sweat_smile: 

```suggestion

```

Reply:
Deleted the whole comment in 7d73d97b